Chatto/sources/Chat Items/ChatItemCompanion.swift

```diff
@@ -28,6 +28,15 @@ public protocol ChatItemsDecoratorProtocol {
     func decorateItems(_ chatItems: [ChatItemProtocol]) -> [DecoratedChatItem]
 }
 
+public final class ChatItemsDecorator:  ChatItemsDecoratorProtocol {
```

Comment:
What do you think about renaming this implementation to something like `DefaultChatItemsDecorator` or `NoopChatItemsDecorator`

To emphasize the fact that it does nothing?